igor-web/src/test/groovy/com/netflix/spinnaker/igor/wercker/WerckerBuildMonitorSpec.groovy

```diff
@@ -0,0 +1,258 @@
+/*
+ * Copyright (c) 2017, 2018, Oracle Corporation and/or its affiliates. All rights reserved.
+ *
+ * The contents of this file are subject to the Apache License Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * If a copy of the Apache License Version 2.0 was not distributed with this file,
+ * You can obtain one at https://www.apache.org/licenses/LICENSE-2.0.html
+ */
+package com.netflix.spinnaker.igor.wercker
+
+import com.netflix.spectator.api.NoopRegistry
+import com.netflix.spinnaker.igor.IgorConfigurationProperties
+import com.netflix.spinnaker.igor.config.WerckerProperties
+import com.netflix.spinnaker.igor.history.EchoService
+import com.netflix.spinnaker.igor.wercker.WerckerService
+import com.netflix.spinnaker.igor.wercker.model.Application
+import com.netflix.spinnaker.igor.wercker.model.Pipeline
+import com.netflix.spinnaker.igor.wercker.model.Run
+import com.netflix.spinnaker.igor.wercker.model.Owner
+import com.netflix.spinnaker.igor.model.BuildServiceProvider
+import com.netflix.spinnaker.igor.service.BuildMasters
+import com.netflix.spinnaker.kork.eureka.RemoteStatusChangedEvent
+import com.netflix.spinnaker.igor.history.model.Event
+import com.netflix.spinnaker.igor.config.WerckerProperties.WerckerHost
+
+import rx.schedulers.TestScheduler
+import spock.lang.Shared
+import spock.lang.Specification
+import spock.lang.Subject
+
+import java.util.Optional
+import java.util.concurrent.TimeUnit
+import java.util.List
+
+import org.slf4j.Logger
+import retrofit.RetrofitError
+import rx.schedulers.Schedulers
```

Comment:
Move these imports up to the same group as `rx.schedulers.TestScheduler` & check sort order.